buildconfig/config_msys2.py

```diff
@@ -494,10 +496,13 @@ def main(auto_config=False):
     # we convert the prebuilt dir to a Windows absolute path.
     # e.g. /mingw64 (MSYS2)  ->  C:/msys64/mingw64 (Windows)
     prebuilt_msys_dir = {
-        'x86': '/mingw32',
-        'x64': '/mingw64'
+        'mingw32': '/mingw32',
+        'mingw64': '/mingw64',
+        'ucrt64':  '/ucrt64',
+        'clang64': '/clang64',
+        'clang32': '/clang32',
     }
-    prebuilt_dir = get_absolute_win_path(prebuilt_msys_dir[machine_type])
+    prebuilt_dir = get_absolute_win_path(prebuilt_msys_dir[MINGW_ARCH])
```

Comment:
```suggestion
    prebuilt_dir = get_absolute_win_path(f"/{MINGW_ARCH}")
```
`prebuilt_msys_dir` can be removed, and this replacement can be added

Reply:
> 

But doesn't pygame need to support python 3.6 whitch doedn't support f-string?

Reply:
pygame needs to support python 3.6 yes, but f-strings were added in 3.6. They are not on 3.5 (which we don't support)